packages/venia-concept/src/components/Checkout/submitButton.js

```diff
@@ -17,8 +17,8 @@ class SubmitButton extends Component {
         const disabled = isDisabled(submitting, valid);
 
         return (
-            <Button disabled={disabled} onClick={submitOrder}>
-                Place Order
+            <Button filled={true} disabled={disabled} onClick={submitOrder}>
```

Comment:
Just for my own education, do we now have four possible button states?

1. Not filled, Not disabled
2. Not filled, disabled
3. Filled, Not disabled
4. Filled, disabled